app.py

```
@@ -2116,25 +2131,130 @@ async def api_weekly_menu(req: WHOTargetsRequest) -> WeeklyMenuResponse:
+            return meals_out
+
+        normalised_daily_menus: List[Dict[str, Any]] = []
+        raw_daily_menus = _safe_attr(week_menu, "daily_menus", []) or []
+        for menu in raw_daily_menus:
+            meals = _normalise_meals(_safe_attr(menu, "meals", []))
+            total_kcal = 0.0
+            for meal in meals:
+                kcal = meal.get("kcal", meal.get("calories", 0))
+                total_kcal += _to_float(kcal, 0.0)
+
+            # Different backends/mocks expose per‑day cost under varying names:
```

Ruff (0.14.0)

Comment contains ambiguous `‑` (NON-BREAKING HYPHEN). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)